index.js

```diff
@@ -23,8 +23,10 @@ app.get('/', function (req, res) {
     return;
   }
 
-  tz['timezone'] = geoTz.tz(tz['lat'], tz['lon']);
-  res.send(JSON.stringify(tz));
+
+timezoneList = find(tz['lat'], tz['lon']);
+tz['timezone'] = timezoneList[0] ? timezoneList[0] : null;
```

Comment:
```suggestion
timezoneList = find(tz['lat'], tz['lon']);
tz['timezone'] = timezoneList[0] || null;
```
or if `undefined` is ok
```suggestion
tz['timezone'] = find(tz['lat'], tz['lon'])[0];
```
😇 🍻